packages/applications-service-api/Dockerfile

```diff
@@ -14,18 +14,23 @@ RUN npm i
 # Stage 2/3: Run tests
 FROM node:14-alpine AS test
 
+WORKDIR /opt/app
+
 COPY --from=build /opt/app ./
 
 COPY packages/applications-service-api/__tests__ ./__tests__
 COPY packages/applications-service-api/jest.config.js jest.config.js
-COPY packages/applications-service-api/.eslintrc .eslintrc
+COPY packages/applications-service-api/.eslint* ./
+COPY packages/applications-service-api/.prettier* ./
 
 RUN npm run lint
-RUN npm run documentation
+RUN npm run documentation; echo $? > /npm.exitcode
```

Comment:
Doesn't this swallow the error?  We want test failures to fail the build

Reply:
Yeah it does, but the pipeline contains a step that publishes the results to Azure DevOps "Test Plans" section. This publishes the results, and fails the build if any test failures.

If this build step fails, then the report step never publishes its results. Hence we swallow the error here :)